### PR TITLE
Feature/services

### PR DESCRIPTION
**Pull request by akmiller01**
Probably all good, but I cannot test since the admin interface appears to be broken...
image

Thought you could use the `skill` section in common with whatwedo and services.

### CODE REVIEW

**Comment by akmiller01** [comment]
Working now! CSS for Work Examples seem to be missing, however:

image

**Comment by edwinmp** [comment]
The git history is broken ...

File: di_website/templates/whatwedo/services_page.html [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+    <section class="section section--alt">
+        <div class="row row--narrow">
+            <h2 class="section__heading">Our skills</h2>
+            <div class="is-typeset max-meter">
+                <p>Our range of skills and expertise enables us to deliver our work:</p>
+            </div>
+            <div class="l-2up">
+                <div class="l-2up__col">
+                    <ul class="accordion">
```
Comment (edwinmp) [inline di_website/templates/whatwedo/services_page.html]: Could you test with at least 4 skills and see if they'll appear in 2 columns...

File: di_website/templates/whatwedo/services_page.html [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+                      </li>
+                      {% endif %}
+                      {% endfor %}
+                  </ul>
+                </div>
+            </div>
+        </div>
+    </section>
+
+{% if self.services_related_news %}
```
Comment (edwinmp) [inline di_website/templates/whatwedo/services_page.html]: Just doing this doesn't work ... I usually go with `.count` i.e `self.services_related_news.count`

File: di_website/templates/whatwedo/services_page.html [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+            <div class="l-2up-3up">
+              {% for item in self.services_related_news.all %}
+                  {% include 'includes/cards/card-news.html' with show_tag=True %}
+              {% endfor %}
+            </div>
+        </div>
+    </section>
+{% endif %}
+
+{% if self.services_related_example %}
```
Comment (edwinmp) [inline di_website/templates/whatwedo/services_page.html]: Same here...

**Pull request merged**